core/config/database_config.go

```diff
@@ -27,4 +50,6 @@ type Database interface {
 	ORMMaxIdleConns() int
 	ORMMaxOpenConns() int
 	TriggerFallbackDBPollInterval() time.Duration
+	LogSQL() bool
+	SetLogSQL(bool)
```

Comment:
I didn't expect to see the `Set` method exposed here but probably nbd :shrug:

Reply:
I think I added it here since I needed to move the LogSQL command into this interface, and it felt like it made sense to move the SetLogSQL command as well.

However, definitely happy to keep them separate (and I think this is less relevant to the general approach I'm trying to illustrate here).

Reply:
My gut is that we don't need to expose the setter to anything that uses config. It should only be called from CLI/UI actions.